src/App/Footer/Footer.tsx

```diff
@@ -15,7 +15,7 @@ function Footer({ linkGroups, socials }: Props) {
         const intercom = document.querySelector(".intercom-launcher");
         intercom?.classList.toggle("hidden", e.isIntersecting);
       },
-      { threshold: [0.5] }
+      { threshold: [0.5] },
```

Comment:
@SovereignAndrey, some formatting of biome could be ignored just like this one: can be turned off via `javascript.formatter.trailingComma = es5`